### PR TITLE
[Admin] Add initial ballerina Admin domain service implementation  

### PR DESCRIPTION
**Pull request by RakhithaRR**
## Purpose
This PR adds the initial implementation of the ballerina service for admin domain service.

Sample config for admin DS will be as follows.

```
[wso2.admin_service.admin.datasourceConfiguration]
description = "APK DB"
url = "jdbc:postgresql://localhost:5432/apkdb"
username = "admin"
password = "admin"
validationTimeout = 30000
testQuery = "SELECT 1"


[wso2.admin_service.admin.throttleConfig]
enableUnlimitedTier = true
```

Related issue: https://github.com/wso2/apk/issues/121

### CODE REVIEW

File: admin/admin-domain-service/build.gradle [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+        group 'ballerina'
+        description 'Build ballerina code'
+        commandLine 'bal', 'build'
+    }
+    
+    task bal_test(type: Exec) {
+        group 'ballerina'
+        description 'Run ballerina tests'
+        commandLine 'bal', 'test'
+    }
```
Comment (binodmx) [inline admin/admin-domain-service/build.gradle:16]: shall we create common ballerina.gradle in common-gradle-scripts and import it here like below?

`apply from: "$rootDir/../common-gradle-scripts/ballerina.gradle"`

File: admin/admin-domain-service/admin-service/build.gradle [hunk 2]
```diff
@@ -0,0 +1,7 @@
+plugins {
+    id 'net.researchgate.release'
+}
```
Comment (binodmx) [inline admin/admin-domain-service/admin-service/build.gradle:3]: shall we remove this as release plugin is added to the root project?

File: admin/admin-domain-service/admin-service/build.gradle [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+
+tasks.named('build').configure {
+    dependsOn 'bal_build'
+}
```
Comment (binodmx) [inline admin/admin-domain-service/admin-service/build.gradle:7]: Shall we replace with this as build is not a default task?
```
tasks.register('build') {
    group 'build'
    description 'Build ballerina component.'
    dependsOn 'bal_build'
}
```

File: admin/admin-domain-service/build.gradle (hunk 1 above)
Comment (RakhithaRR) [reply admin/admin-domain-service/build.gradle]: Fixed with [ca3ec70](https://github.com/wso2/apk/pull/138/commits/ca3ec702a70164c55f4f85fed7694b02286dc96c)

File: admin/admin-domain-service/admin-service/build.gradle (hunk 2 above)
Comment (RakhithaRR) [reply admin/admin-domain-service/admin-service/build.gradle]: Fixed with [ca3ec70](https://github.com/wso2/apk/pull/138/commits/ca3ec702a70164c55f4f85fed7694b02286dc96c)

**Review by binodmx — APPROVED** [review]

**Review by Krishanx92 — APPROVED** [review]

**Review by tharindu1st — APPROVED** [review]

**Pull request merged**